lib/common/common/src/cpu.rs

```diff
@@ -208,7 +208,7 @@ pub enum ThreadPriorityError {
     #[error("Failed to set thread priority: {0:?}")]
     SetThreadPriority(thread_priority::Error),
     #[error("Failed to parse thread priority value: {0}")]
-    ParseNice(&'static str),
+    ParseNice(String),
```

Comment:
Why is this necessary? Due to a dependency bump?

Reply:
I assume it was a dependency bump, yes. But let's see if we can avoid it by not updating the whole project